controllers/articles.controller.js

```diff
@@ -13,6 +13,18 @@ exports.getAllArticles = (req, res, next) => {
    })
 }
 
+exports.updateArticleVotes = (req, res, next) => {
+   const articleNo = req.params
+   const VoteAmount = req.body
+   updateVotes(articleNo, VoteAmount)
+   .then((article) => {
+      res.status(200).send(article)
```

Comment:
Lets make sure to send the result back on an object - you will likely need to update your tests to do this so make sure it fails first 😉

Reply:
That was speedy! Thanks Rich, i'll take a look!